### PR TITLE
Add network information while adding new show

### PR DESCRIPTION
**Pull request by fernandog**
image

### CODE REVIEW

**Comment by p0psicles** [comment]
What does it look like using tmdb or tvmaze?

**Comment by fernandog** [comment]
Doesn't matter which indexer

**Comment by fernandog** [comment]
we use the common field called `network`

image

**Comment by fernandog** [comment]
btw should we order by date? Seems there is no order in the result list

File: medusa/server/web/home/add_shows.py [hunk 1]
```diff
@@ -100,7 +100,8 @@ def searchIndexersForShowName(search_term, lang=None, indexer=None):
 
         for i, shows in iteritems(results):
             final_results.extend({(indexerApi(i).name, i, indexerApi(i).config['show_url'], int(show['id']),
-                                   show['seriesname'].encode('utf-8'), show['firstaired']) for show in shows})
+                                   show['seriesname'].encode('utf-8'), show['firstaired'] or 'N/A',
+                                   show['network'] or 'N/A') for show in shows})
```
Comment (p0psicles) [inline medusa/server/web/home/add_shows.py]: Are there any networks which should be encoded?

Comment (fernandog) [reply medusa/server/web/home/add_shows.py]: I don't know. Same applies for show names right? No issues about that

**Comment by fernandog** [comment]
@p0psicles anything else?

**Pull request merged**